source/architecture.rst

```diff
@@ -1,230 +1,245 @@
+database.
 
-Service Orchestrator
-********************
+flowbroker
+**********
 
-This service provides a high level API for configuring the *dojot* with the objective of reducing the need of knowing
-how to handle each one of the services that are part of the platform. More specifically, it is responsible for modeling
-different services, exposing a simplified configuration interface and propagating this configuration to the various
-services when requested. It acts as a centralized configuration manager for multiple services.
-
-Fiware Perseo
-**************
+This service provides a high level API for configuring the *dojot* with the
+objective of reducing the need of knowing how to handle each one of the
+services that are part of the platform. More specifically, it is responsible
+for modeling different services, exposing a simplified configuration interface
+and propagating this configuration to the various services when requested. It
+acts as a centralized configuration manager for multiple services.
 
-The CEP service is responsible for analysing in real time the data processing flows for selected events and triggering
-actions when specific conditions or thresholds are reached. This component is used for creating notification events from
-the pure data that is incoming from the IoT devices. It is integrated with the platform through the context broker and
-its configuration is abstracted by the service orchestrator.
+History
+*******
 
-History (Logstash)
-******************
+The History component works as a pipeline for data and events that must be
+persisted on a database. The data is converted into an storage structure and is
+sent to the corresponding database.
 
-The Logstash connects to the context broker and works as a pipeline for data and events that must be persisted on a
-database. The data is converted into an storage structure and is sent to the corresponding database.
+For internal storage, the MongoDB non-relational database is being used, it
+allows a Sharded Cluster configuration that may be required according to the
+use case.
 
-For internal storage, the MongoDB non-relational database is being used, it allows a Sharded Cluster configuration that
-may be required according to the use case.
-
-The data may also be directed to databases that are external do the *dojot* platform, requiring only a proper
-configuration of Logstash and the data model to be used.
+The data may also be directed to databases that are external do the *dojot*
+platform, requiring only a proper configuration of Logstash and the data model
+to be used.
 
 Logging and Auditing Service
 ****************************
 
-All the services that are part of the *dojot* platform generate usage metrics of its resources and make then available
-to the logging and auditing service, which process this registers and summarize then based on users and applications.
+All the services that are part of the *dojot* platform generate usage metrics
```

Comment:
This service doesn't exist at the moment, shouldn't it be explained in here?

Reply:
Yes